packages/devtools_app/lib/src/shared/diagnostics/inspector_service.dart

```diff
@@ -42,6 +42,7 @@ abstract class InspectorServiceBase extends DisposableController
           inspectorLibraryUri,
           serviceConnection.serviceManager.service!,
           serviceManager: serviceConnection.serviceManager,
+          isolate: serviceConnection.serviceManager.isolateManager.mainIsolate,
```

Comment:
@matt-ragonese will this break your inspector? Do we need to add an `isolate` param to `InspectorServiceBase` instead of assuming we should use the main isolate here?

Reply:
lets add the isolate parameter (should be an optional named param) to InspectorServiceBase, and then pass in the main isolate there instead of directly on the EvalOnDartLibrary instance.